### ISSUE TITLE
Error with the precompiled library

### ISSUE THREAD

**Issue by csc418q5** [alecjacobson/computer-graphics-ray-tracing#16]
I get the this error when I run `cmake -DCMAKE_BUILD_TYPE=Debug -DHW2LIB_DIR=../lib/debug/linux/ ..`
and then `make`. I am using the ug machines.

```
ug135:~/csc418/computer-graphics-ray-tracing/build% make
[ 14%] Linking CXX executable raytracing
/usr/bin/ld: ../lib/debug/linux/libhw2.a(Plane.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(Sphere.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(Triangle.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(TriangleSoup.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(first_hit.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(viewing_ray.cpp.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: ../lib/debug/linux/libhw2.a(write_ppm.cpp.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
```

**Comment by alecjacobson** [comment]
We're only officially supporting the cdf machines.

(_Did you try adding the `-fPIC` compiler option as the message suggests?_)

**Comment by csc418q5** [comment]
How can I add the -fPIC option?

**Comment by alecjacobson** [comment]
fixed in 2c8eea16bf8ea851026f211138a3d5406726cf18
